Update Jenkinsfile to use external library

surminus opened the pull request.

surminus committed:
Update Jenkinsfile to use external library

issyl0 commented:
I was updating the other Ruby versions to `2.4.2`. I didn't realise there were still versions of `2.2` in the stack.

surminus committed:
Use Ruby 2.2 rather than 2.2.2

Our CI environment does not use 2.2. Sourcing the rbenv_version.sh is
problematic with a Jenkinsfile and I'm too lazy to fix it at the moment.

deanwilson commented:
This tool is no longer in active use and the repository will be archived so we're closing this pull request. Thank you for your interest and involvement in this project.
